docs/source/well_swap/well_swap.rst

```diff
@@ -1,305 +1,305 @@
+
+.. _configuration_wsw:
+*********************
+Prepare configuration
+*********************
+
+The downloaded material is already complete and ready to be launched, however it is still useful to understand how the defined problem was configured. Information related to initial guess, objective functions and the model needs to be specified in main EVEREST configuration file. After downloading tutorial files from :ref:`download_material` this file will be located at ``well_swap/everest/model/wellswap_experiment.yml``.
+
+.. _config_main_wsw:
+Main configuration
+##################
+
+The initial guess for EVEREST is located in the section ``controls`` in main configuration file. This is where we insert the initial priority values for each well and time period, and the duration (in days) of each time period:
+
+.. literalinclude:: ../../../data/drogon/well_swap/everest/model/wellswap_experiment.yml
+   :language: yaml
+   :lines: 19-36
+
+At every iteration EVEREST will randomly perturb the current best priority and duration values and use this information to calculate the direction and step size to propose improved priority and duration values (and therefore improved well swapping schedule) for the next iteration. This means that we need to choose the size of the perturbation, see standard deviation input as ``perturbation_magnitude`` in the ``controls`` section. We also need to choose number of perturbations for each geological realization, see keyword ``perturbation_num`` in ``optimization`` section:
+
+.. literalinclude:: ../../../data/drogon/well_swap/everest/model/wellswap_experiment.yml
+   :language: yaml
+   :lines: 41,43
+
+We also need to specify the name of the objective function in the ``objective_functions`` section. EVEREST will require a file with that same name to be generated by one of the forward model jobs.
+
+.. literalinclude:: ../../../data/drogon/well_swap/everest/model/wellswap_experiment.yml
+   :language: yaml
+   :lines: 38-39
+
+.. note::
+   Using priority control values (for parametrization of discrete decisions) requires additional attention to the step size of the line search of the back-end optimizer. In this tutorial, because priority controls are combined with continuous duration controls, the following settings of the optpp_q_newton optimizer Dakota back-end (default in EVEREST) are chosen:
+   .. literalinclude:: ../../../data/drogon/well_swap/everest/model/wellswap_experiment.yml
+      :language: yaml
+      :lines: 41,47-48
+   These settings are case-dependent, well swapping optimization in other cases may require adjusting this ``max_step`` value.
```

Comment:
This is still an invalid syntax and does not render like expected. You need an empty line before the literalinclude.